src/System/Wlog/Launcher.hs

```diff
@@ -124,3 +127,42 @@ buildAndSetupYamlLogging configBuilder loggerConfigPath = do
 -- See this module description.
 initLoggingFromYaml :: MonadIO m => FilePath -> m ()
 initLoggingFromYaml = buildAndSetupYamlLogging mempty
+
+-- | Initializes logging using given 'FilePath' to logger configurations,
+-- runs the action with the given 'LoggerName'.
+runLoggingWithFile :: MonadIO m => FilePath -> LoggerName -> LoggerNameBox m a -> m a
+runLoggingWithFile filename logName f = do
+    buildAndSetupYamlLogging productionB filename
+    usingLoggerName logName f
+
+-- | Default logging configuration with the "node" logger.
+defaultConfig :: LoggerConfig
+defaultConfig = LoggerConfig
+    { _lcRotation        = Nothing
+    , _lcTermSeverityOut = Nothing    -- default will be used
+    , _lcTermSeverityErr = Nothing    -- default will be used
+    , _lcShowTime        = Any True
+    , _lcShowTid         = Any False
+    , _lcConsoleAction   = mempty
```

Comment:
You should use [`defaultHandleAction`](https://github.com/serokell/log-warper/blob/5783fde3f8cca225e146d08173584dacfe67ed3b/src/System/Wlog/LogHandler/Simple.hs#L67) here instead of `mempty` (just like it's done in `productionB`). Otherwise you won't see output to terminal